package NAlice.NWonderlogs;

import "alice/wonderlogs/protos/megamind_prepared.proto";

import "mapreduce/yt/interface/protos/extension.proto";

message TMegamindPreparedWrapper {
    option (NYT.default_field_flags) = SERIALIZATION_YT;

    optional string Uuid = 1 [json_name = "uuid", (NYT.key_column_name) = "uuid"];
    optional string MessageId = 2 [json_name = "message_id", (NYT.key_column_name) = "message_id"];

    message TValues {
        repeated TMegamindPrepared.TMegamindRequestResponse Values = 3 [json_name = "values", (NYT.column_name) = "values"];
    }

    optional string ValuesBase64 = 3 [json_name = "values", (NYT.column_name) = "values"];
}
